backend/app/usecase/search/search.go

```diff
@@ -33,6 +37,7 @@ func (s Search) Search(query Query, filter Filter) (Result, error) {
 		go func() {
 			result, err := s.searchResource(filter.Resources[i], orders[i], query, filter)
 			if err != nil {
+				s.logger.Error(err)
```

Comment:
Let's add a TODO. We should ideally use instrumentation code instead so that we have a centralized place to organize everything and make changes: https://github.com/short-d/short/blob/master/backend/app/usecase/instrumentation/instrumentation.go.